scu-de/src/fiskaltrust.Middleware.SCU.DE.SwissbitCloudV2/SwissbitCloudV2SCUConfiguration.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace fiskaltrust.Middleware.SCU.DE.SwissbitCloudV2
+{
+    public class SwissbitCloudV2SCUConfiguration
+    {
+        public string TseSerialNumber { get; set; }
+        public string TseAccessToken { get; set; }
+        public bool EnableTarFileExport { get; set; } = true;
+        public virtual string CertificationId { get; set; } = "BSI-K-TR-0490-2021";
```

Comment:
This is not the correct certification ID, right? Can we set this to "TBD"?

Reply:
done